frontend/public/components/node.tsx

```diff
@@ -54,9 +55,10 @@ const Header = props => {
     return null;
   }
   return <ListHeader>
-    <ColHead {...props} className="col-md-4 col-sm-5 col-xs-8" sortField="metadata.name">Node Name</ColHead>
-    <ColHead {...props} className="col-md-2 col-sm-3 col-xs-4" sortFunc="nodeReadiness">Status</ColHead>
-    <ColHead {...props} className="col-md-3 col-sm-4 hidden-xs" sortField="metadata.annotations['machine.openshift.io/machine']">Machine</ColHead>
+    <ColHead {...props} className="col-md-3 col-sm-4 col-xs-8" sortField="metadata.name">Node Name</ColHead>
+    <ColHead {...props} className="col-md-2 col-sm-2 col-xs-4" sortFunc="nodeReadiness">Status</ColHead>
+    <ColHead {...props} className="col-md-3 col-sm-3 hidden-xs" sortField="metadata.annotations['machine.openshift.io/machine']">Machine</ColHead>
+    <ColHead {...props} className="col-md-1 col-sm-3 hidden-xs">Taints</ColHead>
```

Comment:
More recent designs have removed taints from the node table. We should go ahead and rollback this change.